lib/measures/radiance_measure/measure.rb

```diff
@@ -1317,7 +1317,7 @@ def parseResults(t_cmds, t_space_names_to_calculate, t_spaceWidths, t_spaceHeigh
           # iterate over each sensor and combine the views together
           new_hash = {}
 
-          t_radGlareSensorViews[space_name]&.each do |sensor, v|
+          t_radGlareSensorViews[space_name].each do |sensor, v|
```

Comment:
hmm, not sure why this wouldn't work `&.` just verifies that the key exists before executing. I think it has been in Ruby since 2.3. Is someone trying to use this in older versions of ruby?

Reply:
Yeah, looks like the reporting user was using Ruby 2.2 which was used up until OS v2.9.1

 load' :/ruby/2.2.0/gems/openstudio-workflow-1.3.4/lib/openstudio/workflow/util/measure.rb:339:in